ledger-core/virtual/vnlmn/register_context.go

```diff
@@ -8,9 +8,26 @@ package vnlmn
 import (
 	"github.com/insolar/assured-ledger/ledger-core/conveyor/smachine"
 	"github.com/insolar/assured-ledger/ledger-core/reference"
+	"github.com/insolar/assured-ledger/ledger-core/rms/rmsbox"
 	"github.com/insolar/assured-ledger/ledger-core/vanilla/throw"
 )
 
+type rawValue struct {
```

Comment:
better to add/move it to rms package pls

Reply:
Renamed to `rawValueProvider` and moved to `rmsbox`.